meson_options.txt

```diff
@@ -22,6 +22,7 @@ option('plugin_modem_manager', type : 'boolean', value : false, description : 'e
 option('plugin_flashrom', type : 'boolean', value : false, description : 'enable libflashrom support')
 option('plugin_coreboot', type : 'boolean', value : true, description : 'enable coreboot support')
 option('systemd', type : 'boolean', value : true, description : 'enable systemd support')
+option('systemd_root_prefix', type: 'string', value: '', description: 'Directory to base systemd’s installation directories on (defaults to prefix)')
```

Comment:
why not just prefix?

Reply:
systemd allows to change this independently of `prefix` so we need to make it overridable too.

Reply:
Looks like Debian in fact sets to `/` so they would already need to override it here.

Alternately, we could install it to systemd root prefix by default and allow overriding it like we do with `systemdsystemunitdir`. In fact we could drop `systemdsystemunitdir` option since systemd hardcodes it to `${rootprefix}/lib/systemd/system`